app/src/main/java/com/ncnf/main/MainActivity.java

```diff
@@ -17,37 +20,76 @@
 
     private BottomNavigationView navigationBar;
 
+    final FragmentManager fragmentManager = getSupportFragmentManager();
+    private Fragment homeFragment;
+    private Fragment mapFragment;
+    private Fragment feedFragment;
+
+    private Fragment activeFragment;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.activity_main);
 
-        if (savedInstanceState == null) {
-            getSupportFragmentManager().beginTransaction()
-                    .replace(R.id.mainFragmentContainerView, new HomeFragment())
-                    .commit();
+        if(savedInstanceState == null){
+            this.homeFragment = new HomeFragment();
+            this.mapFragment = new MapFragment();
+            this.feedFragment = new FeedFragment();
+            this.activeFragment = this.homeFragment;
         }
 
+        fragmentManager.beginTransaction().add(R.id.mainFragmentContainerView, feedFragment).hide(feedFragment).commit();
+        fragmentManager.beginTransaction().add(R.id.mainFragmentContainerView, mapFragment).hide(mapFragment).commit();
+        fragmentManager.beginTransaction().add(R.id.mainFragmentContainerView, homeFragment).hide(homeFragment).commit();
+
+        fragmentManager.beginTransaction().show(activeFragment).commit();
```

Comment:
Good job on figuring out this part :) I assume this solves the issue of having all three fragments appear on launch?

Reply:
Yes ! Finally

Reply:
gg mate!